feature-libs/checkout/base/styles/components/steps/delivery-method/_delivery-mode.scss

```diff
@@ -1,53 +1,29 @@
 %cx-delivery-mode {
   @include checkout-media-style();
 
-  @include media-breakpoint-down(md) {
```

Comment:
Is this meant for removing 'forVersion'?

Reply:
Yes. i removed the forVersion wrapper and made the scss a little more semantic. I put the media query after the scss for the main selector.